index.js

```diff
@@ -32,7 +32,7 @@ module.exports = (target, opts) => {
 		}
 	} else if (process.platform === 'win32' || isWsl) {
 		cmd = 'cmd' + (isWsl ? '.exe' : '');
-		args.push('/c', 'start', '""');
+		args.push('/c', 'start', '""', '/b');
```

Comment:
`args.push('/c', '/b', 'start', '""');`

Reply:
That’s not proper, because /b is a flag for start, not cmd.

Reply:
Oh, then ignore my comment :).